### PR TITLE
Add mobile Nav Page

### PR DESCRIPTION
**Pull request by frempongdev**
In this Pull Request, I made the following Changes :

- Added the Mobile Navigation Pop-Up Page.
- - Edited index.html to accommodate dynamic changes to mobile page.
- - Added complimentary style classes to style.css file.
- - Added script.js file with function to create dynamic elements.
- - Made changes to correct Linter Errors.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Team 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has lighthouse  linter errors please fix it

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by frempongdev** [comment]
@prolajumokeoni , thank you for your feedback.

Is there a way I can solve the liters error since we have tried all we can?

File: style.css [hunk 1]
```diff
@@ -31,14 +31,61 @@
...
+    margin-right: 35%;
+  }
+
+  .main-nav ul li a {
+    text-decoration: none;
+    color: white;
+    font-size: 2.8rem;
+    font-weight: 600;
+    line-height: 44px;
+  }
```
Comment (jerryowusu) [inline style.css:87]: There are some slight differences between the menu items and that of the figma design as indicated below :point_down: . Kindly check the position, margin, and size and ensure that they match the figma design. 

### Your design

Screenshot from 2023-01-03 15-54-11


### Figma design 

Screenshot from 2023-01-03 15-54-18

File: style.css [hunk 2]
```diff
@@ -31,14 +31,61 @@
...
+    display: none;
+  }
+
+  .bg-style {
+    background-color: rgba(96, 112, 255, 0.7);
+    backdrop-filter: blur(8px);
+    width: 100vw;
+    height: 100vh;
+    position: absolute;
+  }
```
Comment (jerryowusu) [inline style.css:57]: The menu is supposed to be an overlay and not scrollable, however, your mobile nav menu is not fixed hence can be scrolled and dissappear from the screen. This does not give a good user experience. Kindly ensure that you fix this to match the figma design and achieve standard best practices. :wink: 

### Your design when scrolled

Screenshot from 2023-01-03 15-54-52


### Figma design 

Screenshot from 2023-01-03 15-54-18

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi Team :wave: , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost-paul-walker


## Highlights
All linters are passing :heavy_check_mark: 
Close button working as expected :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @frempongdev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

yeasss


## Highlights
All linters are passing :heavy_check_mark: 
Great effort so far :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**